src/basic/fd-util.h

```diff
@@ -161,6 +162,21 @@ char *format_proc_pid_fd_path(char buf[static PROC_PID_FD_PATH_MAX], pid_t pid,
 #define FORMAT_PROC_PID_FD_PATH(pid, fd)                                \
         format_proc_pid_fd_path((char[PROC_PID_FD_PATH_MAX]) {}, (pid), (fd))
 
+static inline int proc_fd_enoent_errno(void) {
```

Comment:
given this is more than a simple type change thing or so, i'd just make this a regular function and not bother with the inline thing. LTO should decide on its own if it inlines this or not.